src/lib/partials/CanvasAlignments.svelte

```diff
@@ -159,12 +162,15 @@
 		</thead>
 		<tbody>
 			{#each credentials as credential}
+				{@const canvasBadge = $canvasSelectedIssuerBadges.find(
+					(cb) => cb.openBadgeId == credential.Credential.CredentialId
+				)}
 				<tr class="bg-white border-b">
 					<th scope="row" class="py-4 px-6 font-medium text-gray-900 whitespace-nowrap">
 						{credential.Credential.Name}
 					</th>
 					<td class="py-4 px-6">
-						{#if alignmentExistsForCredential(credential)}
+						{#if canvasBadge?.alignments?.find((a) => a.targetUrl == alignmentUrlForCredential(credential.Credential.CTID))}
```

Comment:
Could you confirm this change is needed/not needed in ManualAlignments.svelte?

Could we move this into the alignmentExistsForCredential function?